pkg/generator/deploy_tmpl.go

```diff
@@ -54,6 +59,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: OPERATOR_NAME
```

Comment:
Similarly can you make the `OPERATOR_NAME` and `WATCH_NAMESPACE` strings as variables in the template so we can set them using the namespace constants defined in k8sutil/constants.go.
```Go
type OperatorYaml struct {
	ProjectName string
	Image       string
        NameEnv  string
        NamespaceEnv  string
	MetricsPort int
}
```